packages/office-ui-fabric-react/src/utilities/dateValues/DateValues.ts

```diff
@@ -12,10 +12,19 @@ export enum DayOfWeek {
 }
 
 /**
+ * First week of the year settings types
+ */
+export enum FirstWeekOfYear {
+  FirstDay = 0,
+  FirstFullWeek = 1,
+  FirstFourDayWeek = 2
+}
```

Comment:
I assume this maps to the JSMVVM values, correct? Once you commit these changes and we upgrade owa-calendar to use the new version, we should remove the enum from owa-calendar and use this one.

Reply:
yes it is what being passed through settings.

Reply:
make sure you export per our discussion offline so we can reuse in owa-calendar

---
In reply to: [142754811](https://github.com/OfficeDev/office-ui-fabric-react/pull/3001#discussion_r142754811) [](ancestors = 142754811)

Reply:
#Resolved